R/parameters.R

```diff
@@ -150,25 +150,33 @@ unk_check <- function(x) {
 #' @export
 print.parameters <- function(x, ...) {
   x <- tibble::as_tibble(x)
+
   cat("Collection of", nrow(x), "parameters for tuning\n\n")
 
   print_x <- x %>% dplyr::select(identifier = id, type = name, object)
-  print_x$object <- purrr::map_chr(print_x$object, dplyr::type_sum)
+  print_x$object <-
+    purrr::map_chr(
+      print_x$object,
+      ~if (all(is.na(.x))) {"missing"} else {dplyr::type_sum(.x)}
```

Comment:
Should the missing value description read `missing` or `NA`? I'm wasn't sure if `NA` communicates that
something is actually missing rather than "the print method isn't working here." 

Another possibility: use `cli` or another package to color the missing values in the tibble output.

Reply:
I do like the use of "missing" here.